### PR TITLE
Add new wms for wms import tool

### PR DESCRIPTION
**Pull request by pfanguin**

### CODE REVIEW

File: src/js/ImportWmsController.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ goog.provide('ga_importwms_controller');
       defaultWMSList: [
         'https://wms.geo.admin.ch/',
         'http://ogc.heig-vd.ch/mapserver/wms',
+        'http://owsproxy.lgl-bw.de/owsproxy/ows/WMS_Maps4BW?',
```
Comment (oterral) [inline src/js/ImportWmsController.js]: Remove the `?` pls

**Comment by pfanguin** [comment]
it's done

File: src/js/ImportWmsController.js [hunk 2]
```diff
@@ -10,6 +10,7 @@ goog.provide('ga_importwms_controller');
       defaultWMSList: [
         'https://wms.geo.admin.ch/',
         'http://ogc.heig-vd.ch/mapserver/wms',
+        'http://owsproxy.lgl-bw.de/owsproxy/ows/WMS_Maps4BW',
```
Comment (oterral) [inline src/js/ImportWmsController.js]: `https`?

**Comment by pfanguin** [comment]
it does not work with https

**Comment by oterral** [comment]
It works but you need to allow the access in chrome. Ok thanks.

**Pull request merged**